src/moveir/preprocessor/mod.rs

```diff
@@ -327,7 +340,7 @@ impl Visitor for MovePreProcessor {
             );
         }
 
-        if ctx.asset_context.is_some() && enclosing_identifier != "Quartz$Global" {
+        if ctx.asset_context.is_some() && enclosing_identifier != "Flint_Global" {
```

Comment:
Magic strings: Look at this and the difference in the next conditional, if you look at the git diff you can see why we don't like magic strings
**Also, never use `is_some` if you're going to use the value!**